src/core/olimageryprovider.js

```diff
@@ -40,6 +40,17 @@ olcs.core.OLImageryProvider = function(source, opt_fallbackProj) {
 
   this.ready_ = false;
 
+  var proxy = this.source_.get('olcs.proxy');
+  if (goog.isDef(proxy)) {
+    if (goog.isFunction(proxy)) {
+      this.proxy_ = {
+        getURL: proxy
```

Comment:
I think you need to quote the key: `'getURL'` to make sure the name is not minified by closure in advanced mode.